rpc-ipc.md

```diff
@@ -0,0 +1,249 @@
+    span = tracer.start_span(operation_name=operation_name,
+        child_of=parent_span)
+else:
+    span = tracer.start_span(operation_name=operation_name)
+```
+
+If you're using the request-to-span mapping technique, your approach might look like:
+
+```
+parent_span = tracer.get_span(request)
+span = tracer.start_span(
+    operation_name=operation_name,
+    child_of=parent_span)
+```
+
+You can see examples of this approach in [gRPC](https://github.com/grpc-ecosystem/grpc-opentracing/blob/master/java/src/main/java/io/opentracing/contrib/ActiveSpanSource.java) and [JDBI](https://github.com/opentracing-contrib/java-jdbi/blob/9f6259538a93f466f666700e3d4db89526eee23a/src/main/java/io/opentracing/contrib/jdbi/OpenTracingCollector.java#L153).
+
+### Inject the Span
+
+This is where you pass the trace information into the client's request so that the server you send it to can continue the trace. If you're sending an HTTP request, then you'll just use the HTTP headers as your carrier.
+
+span = # start span from the current trace state
+`tracer.inject(span, opentracing.Format.HTTP_HEADERS, request.headers)`
+
+### Finish the Span
+
+When you receive a response, you want to end the span to signify that the client request is finished. Just like on the server side, how you do this depends on how your client request/response processing happens. If your filter wraps the request directly you can just do this:
+
+```
+def filter(request, response):
+    span = # start span from the current trace state
+    tracer.inject(span, opentracing.Format.HTTP_HEADERS, request.headers)
+    response = send_request(request)
+    if response.error:
+       span.set_tag(opentracing., true)
+    span.finish()
+```
+
+Otherwise, if you have ways to process the request and response separately, you might extend your tracer to include a mapping of client requests to spans, and your implementation would look more like this:
+
+```
+def process_request(request):
+    span = # start span from the current trace state
+    tracer.inject(span. opentracing.Format.HTTP_HEADERS, request.headers)
+    tracer.add_client_span(request, span)
+def process_response(request, response):
+    tracer.finish_client_span(request)
+```
+
+## Closing Remarks
+
+Once you've packaged your implementation, email us at [community@opentracing.io](mailto://community@opentracing.io) with your implementation details (platform, description, github username) and we'll create a repo for you under [opentracing-contrib](https://github.com/opentracing-contrib/), so that others will be able to find and use your integration. You can also find there concrete examples of OpenTracing integrations into different open source projects.
+
+If you're interested in learning more about OpenTracing, join the conversation by joining our [mailing list](http://opentracing.us13.list-manage.com/subscribe?u=180afe03860541dae59e84153&id=19117aa6cd) or [Gitter](https://gitter.im/opentracing/public).

```

Comment:
nit: is the mailing list URL intended to have the `u=` and `id=` for stuff like this?